catalog/cataloger_merge.go

```diff
@@ -261,3 +269,7 @@ func (c *cataloger) mergeNonDirect(_ context.Context, _ db.Tx, previousMaxCommit
 	}).Debug("Merge non direct - feature not supported")
 	return ErrFeatureNotSupported
 }
+
+func int64Str(l int64) string {
```

Comment:
too much - I think it will be clear to just see the function you call or just use fmt.Sprintf

Reply:
Removed